.github/workflows/ghcr_core_master.yml

```diff
@@ -1,6 +1,7 @@
 name: Push Core Examples
 
 on:
+  pull_request:
```

Comment:
@EngHabu is this something that's okay to do?  do we really want to push every image on every PR?

Reply:
not ok.. I'm guess @evalsocket added this temporarily to trigger it on the PR for testing... no?

Reply:
it's ok because we have a condition in action for enable/disable push....check https://github.com/flyteorg/flytesnacks/pull/186/files/01c6a3d27caf38e45629d411cbb676ce2ab405d1#diff-ffafe11bbfb408be86a94e2fc82175f029616395e8ee37c19a80c50e89387020R28